File: we_1_5_geometry_sets_the_floor_dop.m
% Integrated Navigation Systems — Chapter 1: Introduction to Integrated Navigation and PNT
% Worked Example 1.5 — Geometry sets the floor: DOP
% Companion code (educational reference implementation; see the book for context).

UERE = 4; HDOP = 1.5;
sigmaH = HDOP * UERE;
fprintf('Horizontal error: %.1f m\n', sigmaH);   % -> 6.0 m

File: we_1_5_geometry_sets_the_floor_dop.py
# Integrated Navigation Systems — Chapter 1: Introduction to Integrated Navigation and PNT
# Worked Example 1.5 — Geometry sets the floor: DOP
# Companion code (educational reference implementation; see the book for context).

UERE, HDOP = 4.0, 1.5
sigmaH = HDOP * UERE
print(f'Horizontal error: {sigmaH:.1f} m')   # -> 6.0 m
